src/api/services/timecardService.js

```diff
@@ -18,3 +18,11 @@ export const getTimecard = async (params) => {
     throw new Error(serviceName + ' getTimecard function threw ' + error);
   }
 };
+
+export const saveTimecard = async (payload, params) => {
```

Comment:
should this be called saveTime**Entry** instead?

Reply:
I think it should, renamed to saveTimeEntry